feature/bgp/policybase/otg_tests/prefix-set_test/README.md

```diff
@@ -0,0 +1,47 @@
+# RT-1.33: BGP Policy with prefix-set matching
+
+## Summary
+
+BGP policy configuration with prefix-set matching
+
```

Comment:
Should add heading for testbed type.  ie: https://github.com/openconfig/featureprofiles/blob/main/doc/test-requirements-template.md#testbed-type

Reply:
added testbed type from https://github.com/openconfig/featureprofiles/tree/main/topologies